worlds/osrs/Names.py

```diff
@@ -0,0 +1,200 @@
+class RegionNames:
+    Lumbridge = "Lumbridge"
+    Lumbridge_Swamp = "Lumbridge Swamp"
+    Lumbridge_Farms_East = "Lumbridge Farms East"
```

Comment:
You see, I thought that at first, but I don't see _any_ other worlds using StrEnum. What does it gain from using one over using values like this?

Reply:
StrEnum is not core Python. What I meant was `class RegionNames(str, Enum)`

Reply:
As for your question, I'm not actually sure tbh, it's not something I could justify, just something I have learned through osmosis & vibes. It might actually not be bad